<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Studentpage</class>
 <widget class="QDialog" name="Studentpage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>203</width>
    <height>333</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="autoFillBackground">
   <bool>true</bool>
  </property>
  <widget class="QPushButton" name="btnAdd">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>80</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{border:none;
color:white;
background-color:rgb(64, 1, 11); 
font:&quot;Arial&quot;; 
border-radius:20px;}
QPushButton:Hover
{color:black;
background-color:rgb(242, 201, 96);
border-radius:50px;}
QPushButton:pressed
{background-color:green;}</string>
   </property>
   <property name="text">
    <string>Add Item</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="lineEditFinder">
   <property name="geometry">
    <rect>
     <x>1100</x>
     <y>160</y>
     <width>271</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextBrowser" name="lineEditName">
   <property name="geometry">
    <rect>
     <x>1100</x>
     <y>110</y>
     <width>271</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btnExitStudent">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>180</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{border:none;
color:white;
background-color:rgb(64, 1, 11); 
font:&quot;Arial&quot;; 
border-radius:20px;}
QPushButton:Hover
{color:black;
background-color:rgb(242, 201, 96);
border-radius:50px;}
QPushButton:pressed
{background-color:green;}</string>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>1010</x>
     <y>210</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>DESCRIPTION</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="textEditDescription">
   <property name="geometry">
    <rect>
     <x>1030</x>
     <y>310</y>
     <width>511</width>
     <height>441</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>1020</x>
     <y>110</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>NAME</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>1020</x>
     <y>160</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>FINDER</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnRetrieve">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>130</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{border:none;
color:white;
background-color:rgb(64, 1, 11); 
font:&quot;Arial&quot;; 
border-radius:20px;}
QPushButton:Hover
{color:black;
background-color:rgb(242, 201, 96);
border-radius:50px;}
QPushButton:pressed
{background-color:green;}</string>
   </property>
   <property name="text">
    <string>Retrieve Item</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>-8</x>
     <y>-1</y>
     <width>221</width>
     <height>341</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(10, 18, 92);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <zorder>label_4</zorder>
  <zorder>btnAdd</zorder>
  <zorder>lineEditFinder</zorder>
  <zorder>lineEditName</zorder>
  <zorder>btnExitStudent</zorder>
  <zorder>label_3</zorder>
  <zorder>textEditDescription</zorder>
  <zorder>label</zorder>
  <zorder>label_2</zorder>
  <zorder>btnRetrieve</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
